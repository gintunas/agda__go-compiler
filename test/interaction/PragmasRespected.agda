{-# OPTIONS --show-implicit #-}

module PragmasRespected where

postulate
  Foo : {A : Set₁} → Set
  Bar : Foo {A = Set}

-- Andreas, 2014-10-20, AIM XX:
-- This test used to check that the --show-implicit option
-- is turned on even if the module is just reloaded from
-- an interface file.
-- However, as we now always recheck when reload,
-- this test should now trivially succeed.
