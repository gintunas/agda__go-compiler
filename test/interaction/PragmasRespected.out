(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "Checked,ShowImplicit")
(agda2-info-action "*All Done*" "" nil)
((last . 1) . (agda2-goals-action '()))
(agda2-status-action "Checked,ShowImplicit")
(agda2-info-action "*Inferred Type*" "Foo {Set}" nil)
(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "Checked,ShowImplicit")
(agda2-info-action "*All Done*" "" nil)
((last . 1) . (agda2-goals-action '()))
(agda2-status-action "Checked,ShowImplicit")
(agda2-info-action "*Inferred Type*" "Foo {Set}" nil)
